proto/booking/reservations/complete_open_reservation.proto

```diff
@@ -26,7 +25,8 @@ message CompleteOpenReservation {
 	holms.types.supply.room_types.RoomType room_type = 8;
 	repeated holms.types.crm.guests.Guest additional_guests = 9;
 	bool tax_exempt = 11;
-	holms.types.booking.reservations.ReservationGuaranteeStatus guarantee_status = 12;
+	// Deprecated guarantee_status
```

Comment:
I'd probably just delete this since we switch versions of this stuff rather synchronously.

It's a lot worse when you have support guarantees for old software in the wild you can't take back.